articles/article-112-compiler.md

```diff
@@ -0,0 +1,32 @@
+<!-- BEGIN TITLE -->
+# We have Machines within Machines
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In 1954, computers just computed numbers. Grace Hopper, a famous
+female programmer around then, had a hard time to convince her
+colleagues that these machines can do more.
+
+The picture is hers and shows how a computer program can translate
```

Comment:
She made the picture below demonstrating how a computer program can translate from one computer language to another. 
- translate from .. to .. 
- I am assuming you mean translating from languages not programs